Nasal/rotor_wash.nas

```diff
@@ -0,0 +1,35 @@
+# Thorsten Renk, modified by HHS
+
+##########################################
+# Preflight control surface check: elevator
+##########################################
+var wash_loop = func {
+
+var vpos = geo.viewer_position();
```

Comment:
Ah,  that was my mistake. Thorsten original Code didn't work,  so I tried another piece